app/src/components/notifications/NotificationBanner.vue

```diff
@@ -51,26 +52,32 @@ switch (props.notification.state) {
 
 <template>
     <div v-if="show" class="inset-x-0 top-0 z-50 text-zinc-900" :class="color">
-        <div class="flex items-center justify-between px-6 py-1">
-            <div class="flex items-center gap-2">
-                <component :is="icon" class="h-5 min-h-5 w-5 min-w-5" />
-                <div class="flex flex-col md:inline-block md:align-middle">
-                    <span class="text-md md:text-sm">{{ notification.title }}</span>
-                    <span v-if="notification.description" class="text-xs md:ml-3">{{
-                        notification.description
-                    }}</span>
+        <component
+            :is="notification.routerLink ? RouterLink : 'div'"
+            :to="notification.routerLink"
+            @click="notification.id ? removeNotification(notification.id) : (show = false)"
```

Comment:
I think this click handler should be moved - if I click the close button, it will also route me to the page, because this also triggers. That's not what I expect.

Perhaps two tests for this could also be added, to both the banner and toast:
- Clicking the link will start a router navigation (this will require a mocked router in the test)
- Clicking the close button while there is a link will only close the notification and not route the user

Reply:
How can we dismiss the notification when clicking on it ?

Reply:
With the close button: 
<img width="72" alt="image" src="https://github.com/user-attachments/assets/d2799c4a-da31-48e6-a5d1-9e38adf83490">

Reply:
Okay.